Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
Looking into this further, I agree we shouldn't patch this with a constant 6 value. There's simply too many cases where this function returns the incorrect range and adding any additional patches just creates a bigger mess. Removing.

---
In reply to: [74478358](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74478358) [](ancestors = 74478358,74473193,74468177,74461625,74359393,74356593,74354002,74342094,74336418)